client/a8c-for-agencies/sections/partner-directory/agency-details/hooks/use-country-list.ts

```diff
@@ -0,0 +1,67 @@
+import { useTranslate } from 'i18n-calypso';
```

Comment:
Could we possibly re-use `client/a8c-for-agencies/sections/signup/agency-details-form/hooks/use-countries-and-states.ts` ?

Reply:
I copied and pasted it, but removed the state part. Really I think we could use it, even with the state part.

Reply:
and the Stripe country check part, I was not sure if that would affect this form with fewer countries.